dotnet/samples/KernelSyntaxExamples/Example22_OpenApiSkill_AzureKeyVault.cs

```diff
@@ -32,11 +32,9 @@ public static async Task RunAsync()
 
     public static async Task GetSecretFromAzureKeyVaultWithRetryAsync(InteractiveMsalAuthenticationProvider authenticationProvider)
     {
-        var retryConfig = new HttpRetryConfig() { MaxRetryCount = 3, UseExponentialBackoff = true };
-
         var kernel = new KernelBuilder()
             .WithLogger(ConsoleLogger.Logger)
-            .Configure(c => c.SetDefaultHttpRetryConfig(retryConfig))
+            .Configure(c => c.SetHttpHandlerFactory(new PollyHttpHandlerFactory<PollyRetryThreeTimesWithBackoff>()))
```

Comment:
I'm curious if retry is needed for this example more often than not.

Reply:
I'm not sure, I just substituted what we had with the Polly impl.